Zappy_IA_Report/DTC/Code/process.py

```
@@ -0,0 +1,17 @@
+def iaProcess(argv: list[str]) -> iaParser:
+        """
+        Main function that handles command line arguments.
+
+        Args:
+            argv (list[str]): Command line arguments
+
+        Returns:
+            None
+        """
+        parser = ia_parser(argv)
+        return parser
```

Ruff (0.11.9)

Unnecessary assignment to `parser` before `return` statement

Remove unnecessary assignment

(RET504)